Participants/Sara Carvalho/hysterectomy_Sara.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml"
+	schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<?xml-model href="../../Schemas/TEILex0-ODD.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="DicEndo">
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>Title</title>
+         </titleStmt>
+         <publicationStmt>
+            <publisher/>
+         </publicationStmt>
+         <sourceDesc>
+            <bibl/>
+         </sourceDesc>
+      </fileDesc>
+   </teiHeader>
+   <text>
+      <body>
+         <entry xml:id="DicEndo.hysterectomy" xml:lang="en">
+         <form type="lemma">
+            <orth>Hysterectomy</orth>
+            <pron><pc>/</pc>ˌhɪstəˈrektəmi<pc>/</pc></pron>
+         </form>
+         <gramGrp>
+            <gram type="pos">noun</gram>
+         </gramGrp>
+            <etym type="inheritance">
+               <cit type="etymon" xml:lang="el"><form><orth>hystéra</orth></form></cit>
```

Comment:
If it's etymology, then it's probably from Ancient and not Modern Greek. And if it's the former, the language code should be "grc".